src/ResourceManager/Compute/Commands.Compute/Generated/VirtualMachineScaleSet/Config/SetAzureRmVmssStorageProfileCommand.cs

```diff
@@ -81,7 +81,7 @@ public partial class SetAzureRmVmssStorageProfileCommand : Microsoft.Azure.Comma
             Mandatory = false,
             Position = 7,
             ValueFromPipelineByPropertyName = true)]
-        public DiskCreateOptionTypes? OsDiskCreateOption { get; set; }
+        public string OsDiskCreateOption { get; set; }
```

Comment:
@hyonholee same comment here, why the switch from enum to string?

Reply:
@cormacpayne This is changed in Swagger spec and client library. They need string for create option because they will keep adding more create option type, like VM sizes.